package/boot/uboot-sunxi/patches/250-sun8i-h3-Nano-Pi-R1-add-device-tree.patch

```diff
@@ -0,0 +1,115 @@
+--- a/arch/arm/dts/Makefile
++++ b/arch/arm/dts/Makefile
+@@ -519,6 +519,7 @@
+ 	sun8i-h2-plus-bananapi-m2-zero.dtb \
+ 	sun8i-h2-plus-libretech-all-h3-cc.dtb \
+ 	sun8i-h2-plus-orangepi-r1.dtb \
++	sun8i-h3-nanopi-r1.dtb \
```

Comment:
That list seems to be sorted. Consider inserting at the correct position.

Reply:
done